source/_components/light.mochad.markdown

```diff
@@ -23,10 +23,13 @@ light:
     devices:
       - address: a1
       - address: a5
+        brightness_levels: 64
```

Comment:
This is an optional configuration variable. We want to keep examples as minimalistic as possible, therefore, please remove this specific change.

Reply:
ok, I updated the branch. The latest version doesn't have this in the example config anymore.